lib/showoff.rb

```diff
@@ -303,25 +304,25 @@ def process_markdown(name, section, content, opts={:static=>false, :pdf=>false,
       # todo: unit test
       lines = content.split("\n")
       @logger.debug "#{name}: #{lines.length} lines"
-      slides = []
-      slides << (slide = Slide.new)
+      slideset = []
```

Comment:
I see why you did this. It would have been easier to read in a separate commit though. What steps did you take to ensure 100% accuracy renaming?